src/Symfony/Component/Notifier/Bridge/Octopush/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/octopush-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony Octopush Notifier Bridge",
+    "keywords": ["sms", "octopush", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Aurélien Martin",
+            "email": "pro@aurelienmartin.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0",
+        "symfony/notifier": "^5.3"
+    },
```

Comment:
Doesn't it requires `symfony/event-dispatcher` in require-dev?

Reply:
No other bridge currently has this, except if they really have a test which tests the event dispatcher integration somehow, I would keep it like this.